modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -1709,5 +1713,24 @@ def applier(df, *args, **kwargs):
         run_and_compare(applier, data=self.data)
 
 
+class TestUnsupportedColumns:
```

Comment:
we already have a class, that stores tests for detecting unsupported columns  ([`TestBadData`](https://github.com/modin-project/modin/blob/6bc748e6e835ecfb6e9779aa93cbd47c2f412511/modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py#L1616)), do we need a new one?

Reply:
that class serves a different purpose from what I see, it checks how bad data is handled. I'm handling here "good" data, or rather I'm checking the _validation_ of is data good or bad...

Reply:
Makes sense